tofhir-engine/src/main/scala/io/tofhir/engine/mapping/MappingTaskExecutor.scala

```diff
@@ -182,7 +188,7 @@ object MappingTaskExecutor {
           case (mappingExpr, resources, fhirInteraction) if fhirInteraction.exists(_.`type` == "patch") && resources.length > 1 =>
             Seq(FhirMappingResult(
               jobId = fhirMappingService.jobId,
-              mappingUrl = fhirMappingService.mappingUrl,
+              mappingTaskName = mappingTaskName,
```

Comment:
Previously, we were using `fhirMappingService.mappingUrl`, which is no longer in use. Can we replace `mappingUrl` with `mappingTaskName` in`fhirMappingService` and continue using it without modifying the method arguments, such as `executeMappingOnInput`?